### PR TITLE
Frontend Refactor

### PR DESCRIPTION
**Pull request by nuudlman**
- Removes component libraries
- More standard file structure
  - Routes are built by App
- Uses BrowserRouter
- Linter is happy

### CODE REVIEW

File: frontend/src/contexts/Auth.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { createContext, useState, ReactNode } from "react";
+
+// Define types
+interface User {
```
Comment (DrewBritt) [inline frontend/src/contexts/Auth.tsx:4]: This should just hold a token

File: frontend/src/contexts/Auth.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    id: string;
+    name: string;
+    // Add other user properties as needed
+}
+
+interface AuthContextType {
```
Comment (DrewBritt) [inline frontend/src/contexts/Auth.tsx:10]: "User" should just be a token

**Review by DrewBritt — CHANGES_REQUESTED** [review]

File: frontend/src/contexts/Auth.tsx (hunk 1 above)
Comment (nuudlman) [reply frontend/src/contexts/Auth.tsx]: Fixed

File: frontend/src/contexts/Auth.tsx (hunk 2 above)
Comment (nuudlman) [reply frontend/src/contexts/Auth.tsx]: Fixed

**Review by DrewBritt — APPROVED** [review]

**Pull request merged**